R/align_dia_runs.R

```diff
@@ -18,6 +18,7 @@
 #' @param peps (integer) ids of peptides to be aligned. If NULL, align all peptides.
 #' @param refRun (string) reference for alignment. If no run is provided, m-score is used to select reference run.
 #' @param applyFun (function) value must be either lapply or BiocParallel::bplapply.
+#' @param saveAlignmentMap (logical) Save a mapping table to track aligned feature ids against reference feature id
```

Comment:
saveAlignedPeaks would be a better name. alignmentMap sounds like saving alignment path through the similarity matrix.

Reply:
done.